viz/TrajectoryVisualization.cpp

```diff
@@ -45,6 +47,11 @@ void TrajectoryVisualization::setColor(const base::Vector3d& color)
     setDirty();
 }
 
+void TrajectoryVisualization::setMaxVelocity(double max_vel)
+{
+    max_velocity = max_vel;
```

Comment:
The OSG nodes won't be updated (this is done by `updateMainNode`). Moreover, `updateMainNode` is called from within the OSG thread, while this method is called from the Qt thread.

The best is to lock (it's less critical than it sounds for something like this, but it's better), with

~~~
boost::mutex::scoped_lock lockit(updateMutex);
~~~

The *required* thing is to call `setDirty()` after you change the parameter, to force vizkit3d to call `updateMainNode`.